### DISCUSSION TITLE
Paper recreations dropped?

### DISCUSSION THREAD

**Discussion by mabdelr** [PMEAL/OpenPNM#2359] [Q&A] [answered]
Hi there,

I know you are making a lot of changes to the repository, however, I was wondering if you dropped the "paper recreation" tutorials? I found them very useful.

Thanks!

**Comment by ma-sadeghi** [comment, marked as answer]
Yes we did, since they were not compatible with v3. However, we plan to make them work with v3 and put them back. They're not lost though, you can find them in the repo history, for instance [here](https://github.com/PMEAL/OpenPNM/tree/v2.8.2). Basically, in the homepage, there's a drop down menu that lets you select the current branch. In it, there's a section called "Tags", which lets you find the OpenPNM pertaining to a particular tag (aka version).

**Reply by mabdelr** [reply]
Got it! Thanks for the help.
